src/useUrlQueryState.ts

```diff
@@ -30,7 +30,12 @@ export const useUrlQueryState = (name: string, overrideInitialValue?: string | n
   return [value, setter];
 };
 
-export const useIntegerUrlQueryState = (name: string, overrideInitialValue?: number): [number | null, (newValue: number | null) => void] => {
-  const [value, setValue] = useUrlQueryState(name, integerToString(overrideInitialValue));
+export const useIntegerUrlQueryState = (name: string, overrideInitialValue?: number, defaultValue?: number): [number | null, (newValue: number | null) => void] => {
```

Comment:
i owuld avoid changing this until we get `useDateUrlQueryState` correct - its hard to read the diff otherwise